sql/src/main/java/io/crate/exceptions/BatchModeWriteOnlyException.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.exceptions;
+
+public class BatchModeWriteOnlyException extends ForbiddenException {
+
+    private static final String MESSAGE = "Only write operations are allowed in Batch mode";
+
+    public BatchModeWriteOnlyException() {
+        super(MESSAGE);
+    }
+
+    @Override
+    public int errorCode() {
+        return 1;
```

Comment:
I think it's okay to just use UnsupportedOperationException. We only send the error messages if an error occurs via postgres protocol

There errorCode wouldn't be used anyway since this is a psql specific error message and there the errorCode isn't used (yet)